### DISCUSSION TITLE
how to set env when docker run  other than when build image

### DISCUSSION THREAD

**Discussion by iamtmoe** [prisma/prisma#10172] [Q&A]
when build a docker image, I need `RUN npx prisma generate`, this command will load .env file in build time, so i need to provide database_url in .env file, is there a way store the databse info on the server other than put into code source file?

**Comment by nenadfilipovic** [comment]
You store it in environment variables where you deploy your app.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
